### DISCUSSION TITLE
thirdSortClick option

### DISCUSSION THREAD

**Discussion by fredleput** [KevinVandy/material-react-table#458] [Q&A] [answered]
### material-react-table version

v1.9.3

### react & react-dom versions

v18.2.0

### Describe the bug and the steps to reproduce it

Hi,

I try to disable the 3rd possibility offered when we sort a column. I would like to sort only in asc and desc direction.
Is there an option available like thirdSortClick ?

Fred.

### Minimal, Reproducible Example - (Optional, but Recommended)

none.

### Do you intend to try to help solve this bug with your own PR?

Yes, I think I know how to fix it and will discuss it in the comments of this issue

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by KevinVandy** [comment, marked as answer]
This is what the `enableSortingRemoval` prop is for

**Reply by fredleput** [reply]
Thx a lot, it works like a charm with `enableSortingRemoval={false}`
Fred.
